VirtualReality/Widgets/qMRMLVirtualRealityView.cxx

```diff
@@ -216,12 +219,14 @@ void qMRMLVirtualRealityViewPrivate::createRenderWindow()
 
   q->updateViewFromReferenceViewCamera();
 
+  this->InteractorStyle->ToggleDrawControls();
```

Comment:
Do you hide the controllers for good? It is often useful to see them. This should be made configurable. I don't think that this should block this merge, so I created this ticket: https://github.com/KitwareMedical/SlicerVirtualReality/issues/30. It would be nice to fix this soon (and make controllers shown by default).

Reply:
This shows them. I didn't mean to leave this in. Will remove.